pkg/operator/ceph/file/controller.go

```diff
@@ -339,18 +340,25 @@ func (r *ReconcileCephFilesystem) reconcile(request reconcile.Request) (reconcil
 				// Set Ready status, we are done reconciling
 				r.updateStatus(r.client, request.NamespacedName, cephv1.ConditionReady, opcontroller.GenerateStatusInfo(cephFilesystem))
 				statusUpdated = true
+				runGoRoutine = true
+			}
+		}
+		// Filesystem performance metrics are new in Ceph Pacific
+		if cephFilesystem.Spec.EnablePerfStats {
```

Comment:
The mirroring health check go routine gathers the metrics? If mirroring is not enabled, does that still make sense?

Reply:
Yes it does, these are performance metrics, not related to the mirroring.